pkgs/servers/home-assistant/custom-components/default.nix

```
@@ -2,5 +2,5 @@
 }:
 
 {
-  prometheus-sensor = callPackage ./prometheus-sensor {};
+  "mweinelt/prometheus_sensor" = callPackage ./mweinelt/prometheus_sensor {};
```

```suggestion
  "mweinelt/prometheus_sensor" = callPackage ./mweinelt-prometheus_sensor {};
```
I don't think we need nested directories here